airbyte-cdk/python/connector_builder/connector_builder_handler.py

```diff
@@ -0,0 +1,33 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+from datetime import datetime
+from typing import Union
+
+from airbyte_cdk.models import AirbyteMessage, AirbyteRecordMessage
+from airbyte_cdk.utils.traced_exception import AirbyteTracedException
+
+
+def list_streams() -> AirbyteRecordMessage:
+    raise NotImplementedError
+
+
+def stream_read() -> AirbyteRecordMessage:
+    raise NotImplementedError
+
+
+def resolve_manifest(source) -> Union[AirbyteMessage, AirbyteRecordMessage]:
+    try:
+        return AirbyteRecordMessage(
+            data={"manifest": source.resolved_manifest},
+            emitted_at=_emitted_at(),
+            stream="",
```

Comment:
other comment disappeared - I think we should set the stream name so the record contains enough information to be interpreted without additional context

Reply:
Makes sense to me, set it to `"resolve_manifest"`.